INPTP_AppForFixing/EmployeeDialog.cs

```diff
@@ -1,58 +1,58 @@
 ﻿using System;
-using System.Linq;
 using System.Windows.Forms;
 
 namespace INPTP_AppForFixing
 {
     public partial class EmployeeDialog : Form
     {
         private MainForm main;
-        private bool employee = false;
+        private bool isEmployee;
```

Comment:
Names with verbs should be used for methods. For variables/attributes are preferred nouns.

Reply:
fixed